runtime/android/app/src/main/cpp/kaldi-native-fbank/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(csrc)
```

Comment:
Is it possible to use `cmake/kaldi-native-fbank.cmake` to download it during building?

Reply:
Yes, I will try ExternalProject_Add function to download and process kaldi-native-fbank project.

Reply:
Fixed